packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -9,18 +9,42 @@ import { MenuItem, MenuItemRole } from "./menu-item";
 export const MenuItemTemplate = html<MenuItem>`
     <template
         role="${x => x.role}"
+        aria-haspopup="${x =>
+            x.submenuElements && x.submenuElements.length > 0 ? "menu" : void 0}"
         aria-checked="${x => (x.role !== MenuItemRole.menuitem ? x.checked : void 0)}"
         aria-disabled="${x => x.disabled}"
         aria-expanded="${x => x.expanded}"
         @keydown="${(x, c) => x.handleMenuItemKeyDown(c.event as KeyboardEvent)}"
         @click="${(x, c) => x.handleMenuItemClick(c.event as MouseEvent)}"
         class="${x => (x.disabled ? "disabled" : "")} ${x =>
             x.expanded ? "expanded" : ""}"
+        ${children({
+            property: "submenuElements",
+            filter: elements("[slot=submenu]"),
+        })}
     >
         ${startTemplate}
         <span class="content" part="content">
             <slot></slot>
         </span>
         ${endTemplate}
+        ${when(
+            x => x.expanded,
+            html<MenuItem>`
+                <fast-anchored-region
+                    :anchorElement="${x => x}"
+                    vertical-positioning-mode="dynamic"
+                    vertical-default-position="bottom"
+                    vertical-inset="true"
+                    horizontal-positioning-mode="dynamic"
+                    horizontal-default-position="end"
+                    class="submenu-region"
```

Comment:
This will likely need a part as well I presume?

Reply:
added part="submenu-region"